src/main/java/org/gridsuite/loadflow/server/service/LoadFlowResultContext.java

```diff
@@ -66,11 +66,11 @@ public static LoadFlowResultContext fromMessage(Message<String> message, ObjectM
     }
 
     @Override
-    protected Map<String, String> getSpecificMsgHeaders() {
+    protected Map<String, Object> getSpecificMsgHeaders() {
         return Map.of(
                 HEADER_LIMIT_REDUCTION,
-                runContext.getLimitReduction() != null ?
-                        runContext.getLimitReduction().toString() :
+                getRunContext().getLimitReduction() != null ?
```

Comment:
Not really related but in line 48 and 51, using test by headers.containsKey() is not enough, the value can be null, pls correct line 48 and 51 as following way:
String limitReductionStr = headers.get(HEADER_LIMIT_REDUCTION);
Float limitReduction = limitReductionStr != null ? Float.parseFloat(limitReductionStr) : null

Reply:
Done. Damn null parameters in map.of won't get me again.